codeforamerica/vita-min
# Fix calculation for line 27 on ID-40

## Link to pivotal/JIRA issue
[- e.g. https://codeforamerica.atlassian.net/browse/ABCD-123](https://codeforamerica.atlassian.net/browse/FYST-1347?atlOrigin=eyJpIjoiNWU3OWI0MTM5M2VkNDc3MmE5OTUwMTk5ZDFhMTAxYTQiLCJwIjoiaiJ9)
## Is PM acceptance required? (delete one)
- Yes - don't merge until JIRA issue is accepted!

**Reminder**: merge main into this branch and get green tests before merging to main
## What was done?
- Fix calculation for line 27; line 27 should be line 21 - line 26 but since the calculation converts line 21 to an integer in the line 27 calculation and doesn't round at the end for the pdf the values come out a little off
## How to test?
- set up an ID-40 case where line 21 is a value where the decimal value rounds up
- check that line 27 = line 21 - line 26
